### ISSUE TITLE
reconstruction loss won't decrease

### ISSUE THREAD

**Issue by billy800413** [auspicious3000/autovc#54]
Hi, thanks for your code.
But after testing it with your test data(p225, p226, p227, p228), reconstruction loss will be stuck around 0.002.(both G/loss_id and G/loss_id_psnt), num_iters is 50000, others keep original.
can you give me any suggestions?

**Comment by Trebolium** [comment]
@billy800413 did you figure this out in the end? Vaguely recall that when i trained at 100k iterations on original test data as described in the paper, it does actually converge to 0.001. However upon training a model using an adaption of the recently uploaded training code, trained on 20 speakers, it lingers around the 0.005 mark. Have you managed to improve your sany further since writing this?
